src/chrome/cdtpDebuggee/features/notes.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this needed?

Reply:
Whoops... Didn't mean to commit this file... Thanks for flagging this :)